pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -245,69 +233,6 @@ func (nc *NodeController) SyncCloudNodes() error {
 	return nil
 }
 
-// SyncProbedNodeStatus synchronizes cluster nodes status to master server.
-func (nc *NodeController) SyncProbedNodeStatus() error {
-	nodes, err := nc.kubeClient.Nodes().List()
-	if err != nil {
-		return err
-	}
-	nodes, err = nc.PopulateNodesStatus(nodes)
-	if err != nil {
-		return err
-	}
-	for _, node := range nodes.Items {
-		// We used to skip updating node when node status doesn't change, this is no longer
-		// useful after we introduce per-probe status field, e.g. 'LastProbeTime', which will
-		// differ in every call of the sync loop.
-		glog.V(2).Infof("updating node %v", node.Name)
-		_, err = nc.kubeClient.Nodes().Update(&node)
-		if err != nil {
-			glog.Errorf("error updating node %s: %v", node.Name, err)
-		}
-	}
-	return nil
-}
-
-// PopulateNodesStatus populates node status for given list of nodes.
-func (nc *NodeController) PopulateNodesStatus(nodes *api.NodeList) (*api.NodeList, error) {
-	var wg sync.WaitGroup
-	wg.Add(len(nodes.Items))
-	for i := range nodes.Items {
-		go func(node *api.Node) {
-			node.Status.Conditions = nc.DoCheck(node)
-			if err := nc.populateNodeInfo(node); err != nil {
-				glog.Errorf("Can't collect information for node %s: %v", node.Name, err)
-			}
-			wg.Done()
-		}(&nodes.Items[i])
-	}
-	wg.Wait()
-	return nc.PopulateAddresses(nodes)
-}
-
-// populateNodeInfo gets node info from kubelet and update the node.
-func (nc *NodeController) populateNodeInfo(node *api.Node) error {
-	nodeInfo, err := nc.kubeletClient.GetNodeInfo(node.Name)
-	if err != nil {
-		return err
-	}
-	for key, value := range nodeInfo.Capacity {
-		node.Status.Capacity[key] = value
-	}
-	if node.Status.NodeInfo.BootID != "" &&
-		node.Status.NodeInfo.BootID != nodeInfo.NodeSystemInfo.BootID {
-		ref := &api.ObjectReference{
-			Kind:      "Minion",
-			Name:      node.Name,
-			UID:       node.UID,
-			Namespace: api.NamespaceDefault,
-		}
-		nc.recorder.Eventf(ref, "rebooted", "Node %s has been rebooted", node.Name)
-	}
-	node.Status.NodeInfo = nodeInfo.NodeSystemInfo
-	return nil
-}
-
 // DoCheck performs various condition checks for given node.
 func (nc *NodeController) DoCheck(node *api.Node) []api.NodeCondition {
```

Comment:
Right, it was used in the test, but looking carefully in it, the test was testing this function:) Removed it and whole its subtree.